Makefile

```diff
@@ -1,6 +1,13 @@
 all: build
 .PHONY: all
 
+# Include the library makefile
+include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
+	golang.mk \
+	targets/openshift/images.mk \
+	targets/openshift/deps.mk \
+)
+
 build:
```

Comment:
you already get build from golang.mk that's why you are seeing:
```
$ make help
Makefile:12: warning: overriding recipe for target `build'
vendor/github.com/openshift/build-machinery-go/make/targets/golang/build.mk:13: warning: ignoring old recipe for target `build'
The following make targets are available:
```

Reply:
Right.  But is that also why I see:

```
Makefile
Makefile:12
```

below?

Reply:
This error is in every make invocation, not specific to `make help`. Because `make help` is parsing the output of `make fake target` it wrongly parses the error as a target but when you fix the error you won't see it in `make help`.